### DISCUSSION TITLE
Obsidian plugins we'd like to support (is it possible and how?)

### DISCUSSION THREAD

**Discussion by olayway** [flowershow/flowershow#683] [General and Help]
## Some nice Obsidian community plugins

- [Obsidian Kanban](https://github.com/mgmeyers/obsidian-kanban)
- [Folder Notes](https://github.com/xpgo/obsidian-folder-note-plugin)

**Comment by ErtugrulSener** [comment]
What also would be nice:
- [Admonition](https://github.com/valentine195/obsidian-admonition)
- [Spaced Repition](https://github.com/st3v3nmw/obsidian-spaced-repetition)

**Comment by rufuspollock** [comment]
@ErtugrulSener we now support [obsidian callouts](https://flowershow.app/docs/callouts) so would admonition be needed as well?

For spaced repetition what would you want on the published site from spaced repetition (interactive support for users?)

**Comment by ErtugrulSener** [comment]
@rufuspollock I think there are some main differences between the callout boxes and Admonition, but the most important one for me is: Syntax.

just feels more natural as you type. Especially try nesting obsidian callouts with the given syntax and you will see, how unreadable it got. Also the design (as far as I can see), including the little icons on the left based on the type of the box is different. That may be subjective, but I do like it more.

Some other features are:
- Collapse (with collapse: true) we can collapse the box
- Color (we can change the color)
- Custom Admonitions
- The copy button

**Comment by rufuspollock** [comment]
@ErtugrulSener to check the current [obsidian-admonitions plugin](https://github.com/valentine195/obsidian-admonition) (have i got the right one) says:

> With this change, Admonitions will be transitioning to a plugin that enhances the core callout box feature.

So it sounds like they are converging on obsidian callouts is that right?

**Comment by ErtugrulSener** [comment]
> @ErtugrulSener to check the current [obsidian-admonitions plugin](https://github.com/valentine195/obsidian-admonition) (have i got the right one) says:
> 
> > With this change, Admonitions will be transitioning to a plugin that enhances the core callout box feature.
> 
> So it sounds like they are converging on obsidian callouts is that right?

I think you missunderstood the description. Obsidian 0.14 is out since some time. The text you quoted says, that they are an enhancememt for obsidian callouts, since the original syntax is shit.
